loader/include.go

```diff
@@ -104,17 +104,27 @@ func ApplyInclude(ctx context.Context, configDetails types.ConfigDetails, model
 			if s, err := os.Stat(f); err == nil && !s.IsDir() {
 				r.EnvFile = types.StringList{f}
 			}
+		} else {
+			envFile := []string{}
+			for _, f := range r.EnvFile {
+				f := filepath.Join(configDetails.WorkingDir, f)
+				if s, err := os.Stat(f); err == nil && !s.IsDir() {
```

Comment:
if explicitly selected env file is a Dir, then this is an error. We only ignore when selecting the _default_ `.env` file.